Small Compiler warning in subsys/fs/shell.c:381:23 in latest release, need argument change only

In file included from /home/robocom/zephyrproject/zephyr/subsys/fs/shell.c:10:
/home/robocom/zephyrproject/zephyr/subsys/fs/shell.c: In function 'cmd_cat':
/home/robocom/zephyrproject/zephyr/subsys/fs/shell.c:381:23: warning: format '%d' expects argument of type 'int', but argument 5 has type 'ssize_t' {aka 'long int'} [-Wformat=]
  381 |    shell_error(shell, "Failed to read from file %s (err: %d)",
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  382 |     path, read);
      |           ~~~~         
      |           |
      |           ssize_t {aka long int}
/home/robocom/zephyrproject/zephyr/include/shell/shell.h:997:34: note: in definition of macro 'shell_error'
  997 |  shell_fprintf(_sh, SHELL_ERROR, _ft "\n", ##__VA_ARGS__)
      |                                  ^~~
/home/robocom/zephyrproject/zephyr/subsys/fs/shell.c:381:59: note: format string is defined here
  381 |    shell_error(shell, "Failed to read from file %s (err: %d)",
      |                                                          ~^
      |                                                           |
      |                                                           int
      |                                                          %ld

--- a/subsys/fs/shell.c
+++ b/subsys/fs/shell.c
@@ -378,7 +378,7 @@ static int cmd_cat(const struct shell *shell, size_t argc, char **argv)
 		}
 
 		if (read < 0) {
-			shell_error(shell, "Failed to read from file %s (err: %d)",
+			shell_error(shell, "Failed to read from file %s (err: %zd)",
 				path, read);
 		}
 

